lib/src/components/NxBinaryDonutChart/NxBinaryDonutChart.scss

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-colors';
+
+.nx-binary-donut-chart-circle {
+  fill: none;
+  transform: rotate(-90deg);
+
+  &:nth-of-type(1) {
```

Comment:
nth child selectors are to be avoided where possible, as they are fragile.  What I'd rather see here is a class for the svg itself (`nx-binary-donut`) which could probably house the fill and transform, and then meaningful classes for the two circles (`nx-binary-donut__background` and `nx-binary-donut__arc` maybe) which set their stroke color.

Reply:
> `nx-binary-donut__background` and `nx-binary-donut__arc` maybe

Changed to use these classes instead of nth child selectors https://github.com/sonatype/sonatype-react-shared-components/pull/172/commits/0b2fad02e32cf9927810950d59c52abe24ac9a2f